data_steward/analytics/cdr_ops/rdr_export_qc.py

```diff
@@ -61,6 +61,28 @@
 '''
 pd.read_gbq(query, dialect='standard')
 
+# ## ID range check
+# Combine step may break if any row IDs in the RDR are larger than the added constant(1,000,000,000,000,000).
+# Rows that are greater than 999,999,999,999,999 the will be listed out here.
+
+domain_table_list = [
+    'condition_occurrence', 'device_exposure', 'drug_exposure', 'location',
+    'measurement', 'note', 'observation', 'procedure_occurrence', 'provider',
+    'specimen', 'visit_occurrence'
+]
+for table in domain_table_list:
```

Comment:
Would that be a good idea to combine all queries together since they return results in the same structure?